screens/RetailOrderFormII.js

```diff
@@ -0,0 +1,1356 @@
+import React from 'react'
+import {reduxForm, Field} from 'redux-form'
+import {Alert, ScrollView, Text, TouchableOpacity, View, StyleSheet} from 'react-native'
+import {connect} from 'react-redux'
+import {Accordion, List} from '@ant-design/react-native'
+import {getLables, getProducts, clearOrder, getfetchOrderInflights, getOrder, getOrdersByDateRange, getTimeDifference} from '../actions'
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome'
+import styles, {mainThemeColor} from '../styles'
+import {LocaleContext} from '../locales/LocaleContext'
+import LoadingScreen from "./LoadingScreen";
+import BackendErrorScreen from "./BackendErrorScreen";
+import {api, dispatchFetchRequest, dispatchFetchRequestWithOption, successMessage, warningMessage, apiRoot} from '../constants/Backend'
+import {ThemeContainer} from "../components/ThemeContainer";
+import {StyledText} from "../components/StyledText";
+import {ListItem} from "react-native-elements";
+import {withContext} from "../helpers/contextHelper";
+import {compose} from "redux";
+import OrderItemDetailEditModal from './OrderItemDetailEditModal';
+import OrderTopInfo from "./OrderTopInfo";
+import DeleteBtn from '../components/DeleteBtn'
+import NavigationService from "../navigation/NavigationService";
+import {handleDelete, handleOrderSubmit, handleRetailCheckout, revertSplitOrder, handlePrintWorkingOrder, handlePrintOrderDetails, handleOrderAction, getTableDisplayName} from "../helpers/orderActions";
+import {SwipeRow} from 'react-native-swipe-list-view'
+import ScreenHeader from "../components/ScreenHeader";
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
+import {SecondActionButton} from "../components/ActionButtons";
+import {printMessage} from "../helpers/printerActions";
+import DropDown from "../components/DropDown";
+import TimeAgo from 'javascript-time-ago';
+import en from 'javascript-time-ago/locale/en';
+import {NavigationEvents} from "react-navigation";
+import {SplitBillPopUp} from '../components/PopUp'
+import SockJsClient from 'react-stomp';
+
+class RetailOrderFormII extends React.Component {
+    static navigationOptions = {
+        header: null
+    }
+    static contextType = LocaleContext
+
+    constructor(props, context) {
```

Comment:
Please remove all the code that is not related to retail mode including submit order, deliver order etc etc.